components/blitz/README.txt

```diff
@@ -1,11 +1,11 @@
 OmeroBlitz
-http://www.openmicroscopy.org/site/support/omero5.2/developers/server-blitz.html
-------------------------------------------------------------------------------
+https://docs.openmicroscopy.org/omero/5.3.3/developers/server-blitz.html
```

Comment:
Do we need to use some sort of version variable here rather than hardcoding the version and needing to update it? Or could we just use /latest/ throughout?